### DISCUSSION TITLE
Default ingress route renaiming api-gateway

### DISCUSSION THREAD

**Discussion by graz-dev** [mia-platform/community#432] [Product Feedback]
### What product would you like to leave feedback for?

Console

### What Console version are you using?

Mia-Platform v12.2.0

### Description

While creating a new project, I noticed that after renaming an API gateway differently from the default proposed by the installation wizard, my endpoints became unreachable (in my case, I created a simple API Portal).

Upon investigation, I realized that in the console configuration, the rule of the ingress route was redirecting traffic to a service named "api-gateway" on port 8080. In my case, the service was named "api-gateway-demo," causing my endpoints to return a 400 error. Modifying this rule in the ingress route file resolved the issue.

It might be helpful to at least warn in the API gateway creation wizard that one could encounter this problem?

![Progetto senza titolo](https://github.com/mia-platform/community/assets/62215881/dbfc7db7-6e8f-4490-a1f7-8d71d32080de)

**Comment by fredmaggiowski** [comment]
Hi @graz-dev, thanks for the feedback. 

Sadly there is little we can do at the moment. the `IngressRoute` resource is not managed by the Console yet, as there is no integration with any IngressController. 

For this reason the Console is not aware on how you are managing your namespace exposition to the outside world, you may be using Traefik (as in this case) or pretty much anything else; adding a warning in the Console when dealing with the API Gateway configuration could lead to misunderstandings for final users that may have a different infrastructure than ours.

We are planning a new feature where users will be able to handle all this custom resource files within the Design area in order to make such changes more transparent to the users, I still don't think there will be a way to provide such insights to the user but at least you'll have to manage the resource along side the API Gateway itself and minimize the human error.
